src/components/core/PivotTable.tsx

```diff
@@ -51,6 +81,28 @@ export interface IGridParams {
     api: IGridApi;
 }
 
+export interface IGridCell {
+    [key: string]: any;
+    data: {
+        [key: string]: any;
+        drillItemMap: {
+            [key: string]: any;
+        };
+    };
+}
+
+// This is a minimal interface for AG Grid's renderer props. Run Debugger to see full object.
+// Imported styles from AG Grid dont seem compatible.
+export interface ILoadingRendererProps {
```

Comment:
all the interfaces into interfaces folder please

Reply:
I think you can find interface for this as well. AG-grid is written in TS, so the should have all the interfaces covered